core/src/main/java/bio/terra/pearl/core/dao/survey/AnswerMappingDao.java

```diff
@@ -25,4 +28,16 @@ public List<AnswerMapping> findBySurveyId(UUID surveyId) {
     public void deleteBySurveyId(UUID surveyId) {
         deleteByProperty("survey_id", surveyId);
     }
+    public Optional<AnswerMapping> findByTargetField(UUID surveyId, AnswerMappingTargetType targetType, String targetFieldName) {
```

Comment:
nit: add blank line between methods
```suggestion

    public Optional<AnswerMapping> findByTargetField(UUID surveyId, AnswerMappingTargetType targetType, String targetFieldName) {
```